portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,35 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
     <script src="script.js"></script>
+    <style>
+    body{
+        font: 22px 'Quicksand', sans-serif;
+        background-color: #a8d0e6;
+    }
+    .center{
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+    }
+    </style>
   </head>
   <body>
+    <div id="navbar">
+        <a class="active" id="home" href="#Home" >Home</a>
+        <a id="background" href="/background.html" >Background</a>
+        <a id="photogal" href="#PhotoGallery">Photo Gallery</a>
+        <a id="blog" href="/blog.html">Blog</a>
+        <a id="contact" href="#Contact">Contact</a>
+    </div>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1>Welcome to Oyin's Portfolio</h1>
+        <h2> Here you will get to know who Oyin Sholeye at Google is</h2>
```

Comment:
nit: remove space after "\<h2\>"

(Sorry for so much nit picks, usually we have a tool that fixes these issues for us 😄 )